packages/now-cli/test/integration.js

```diff
@@ -2203,6 +2247,7 @@ test('create zero-config deployment', async t => {
   const data = JSON.parse(text);
 
   t.is(data.error, undefined, JSON.stringify(data, null, 2));
+  t.truthy(Array.isArray(data.builds), JSON.stringify(data, null, 2));
```

Comment:
I think sourceless deployments will stop assigning `builds` soon

Reply:
@styfle But this is not a sourceless deployment, is it?

Reply:
I think all deployments are sourceless now. But it looks like `builds: []` for sourceless since it no longer updates the `builds` property on the deployment doc for zero config.

I guess I'm not sure why this assertion was added here? Is this test affected by domains?